src/main/java/com/elmc/booking/adapters/incoming/rest/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,129 @@
+package com.elmc.booking.adapters.incoming.rest;
+
+import com.elmc.booking.domain.reservation.exceptions.*;
+import com.elmc.booking.domain.screening.exceptions.*;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@Slf4j
+@ControllerAdvice
+public class ControllerExceptionHandler {
+
+    @ExceptionHandler
+    public ResponseEntity<ErrorMessage> invalidScreeningTimeIntervalExceptionHandler(InvalidScreeningTimeIntervalException exception) {
+        log.error("Exception thrown InvalidScreeningTimeIntervalException: {}", exception.getMessage(), exception);
```

Comment:
Kiedy dobrze jest logować na error a kiedy na warn?

Reply:
Error do błędów, które uniemożliwiają poprawne funkcjonowanie aplikacji, natomiast warn gdy coś jest nie tak, ale aplikacja działa prawidłowo. Tutaj można by to pozmieniać na warny.

Reply:
Dla mnie fajną regułą kciuka jest czy człowiek (administrator?) musi reagować - jak tak to error, jak nie to warn.